openml/datasets/functions.py

```diff
@@ -333,6 +333,28 @@ def _load_features_from_file(features_file: str) -> Dict:
         return xml_dict["oml:data_features"]
 
 
+def _expand_parameter(parameter):
+    expanded_parameter = []
+    if isinstance(parameter, str):
+        expanded_parameter = [x.strip() for x in parameter.split(",")]
+    elif isinstance(parameter, list):
+        expanded_parameter = parameter
+    return expanded_parameter
+
+
+def _validated_data_attributes(attributes, data_attributes, parameter_name):
+    if attributes is not None:
+        for attribute_ in attributes:
+            is_row_id_an_attribute = any([attr[0] == attribute_ for attr in data_attributes])
```

Comment:
The variable name does not make a lot of sense. It just indicates of the attribute is actually an attribute of the dataset, right?